lib/premailer/rails/css_helper.rb

```diff
@@ -36,14 +34,35 @@ def css_urls_in_doc(doc)
         end
       end
 
+      def load_css_with_cache(url)
+        self.cache[url] ||= load_css(url)
+      end
+
+      def cache_enabled?
+        defined?(::Rails) && ::Rails.env.production?
```

Comment:
@fphilipe I think it makes sense to only use cache in production (before it was !development, which affected test). WDYT?